src/plugins/intel_gpu/src/graph/reorder.cpp

```diff
@@ -163,10 +163,7 @@ layout reorder_inst::calc_output_layout(reorder_node const& node, kernel_impl_pa
         // TODO Shouldn't transform be called every time ifmt != ofmt?
         return layout(odt, ofmt, input_layout.get_tensor().transform(ofmt, 1), op);
     } else {
-        if (input_layout.is_static())
-            return layout(odt, ofmt, input_layout.get_tensor(), op);
-        else
-            return layout(input_layout.get_partial_shape(), odt, ofmt, op);
+        return layout(input_layout.get_partial_shape(), odt, ofmt, op);
```

Comment:
I think such a change should be verified through benchmark

Reply:
Yes, after finalize the PR to the state of green CI, all changes will be verified through benchmark